control_group_msgs all_group_controls

-----
state_group_msgs all_group_states